protoflo/__main__.py

```diff
@@ -15,7 +15,7 @@ def register (user_id, label, ip, port):
 	data = {
 		'type': 'protoflo', 
 		'protocol': 'websocket',
```

Comment:
I'm not completely sure, but if this used to work, then I'm guessing, yes.   The result when I tried to register the runtime was `ws://ws://protoflo`